packages/allure-mocha/src/index.ts

```diff
@@ -1,3 +1,8 @@
 import { MochaAllureReporter } from "./MochaAllureReporter";
+import { AllureContainer } from "allure-js-commons";
+import { MochaAllure } from "./MochaAllure";
 
-export = MochaAllureReporter;
```

Comment:
What's the reason to change `export = ` to `module.exports =`? It is the same output in Javascript, but `export = ` also generates proper .d.ts

Reply:
The main reason is in additional exports. TS doesn't allow to do it if we have anything else here:
![image](https://user-images.githubusercontent.com/6638780/80920929-a9f32700-8d7b-11ea-876d-397730571964.png)

Reply:
Right, because you can't mix common js and es exports in the same file. Basically the last `module.exports =` assignment erases all exports above.

This is the reason why the current master branch uses a separate `runtime` file to get around this.

Reply:
So if a leave only a reporter export here, then how would I export a static `allureContainer` from within `MochaAllureReporter`? Or I'd need to move it outside a class the way it was implemented before, and then export via getter in runtime.js?

Reply:
Ok, figured it out. Will push updates soon.